### ISSUE TITLE
Random forest classifier takes too long

### ISSUE THREAD

**Issue by pcountryman** [pcountryman/DOTA_buff_predictor#1]
The random forest classifier has *two* known issues issues:
* takes a long time to run due to various parameter varied
* has poor results

Currently, the **training set always returns perfect scores**, while the test set is a poor performer.

I don't have enough time to diagnose the training issue right now, so I'm going to just comment out the RFC for the time being, and note in the code the training set issue so it can be diagnosed at a later time.
